commune_app/all_models/Vote.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from .Chore import Chore
+from .users import User
+
+
+class Vote(models.Model):
+    id = models.IntegerField(unique=True, primary_key=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    chore = models.ForeignKey(Chore, on_delete=models.CASCADE)
+    approve = models.BooleanField()
+
+    # creates a new vote while checking if the related Chore can be passed or not
+    @staticmethod
+    def create_new_vote(Vid, VotingUser, VotedChore, VoteBool):
```

Comment:
Function names should be lowercase, with words separated by underscores as necessary to improve readability.
Variable names follow the same convention as function names.
for more details you can see: 
https://peps.python.org/pep-0008/#function-and-variable-names

```suggestion
    def create_new_vote(vid, voting_user, voted_chore, vote_bool):
```